Dev/VideoChatClient/Input/AudioInput.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <Windows.h>
+#include <SFML/Audio.hpp>
+
+#include "IInput.hpp"
+#include "VideoChatClientException/VideoChatClientException.hpp"
+
+class AudioInput : public IInput
+{
+public:
+	AudioInput();
+	void take_input() override;
+
+private:
+	void _write_sound_buffer_to_memory(const sf::SoundBuffer& sound_buffer);
+
+private:
+	static constexpr uint32_t SAMPLING_DURATION = 5000;
```

Comment:
Sampling duration should not be constexpr, it should be given to the ctor so it would be easily configurable.

Reply:
Done,